editor/import/resource_importer_layered_texture.cpp

```diff
@@ -544,3 +475,82 @@ ResourceImporterLayeredTexture::ResourceImporterLayeredTexture() {
 
 ResourceImporterLayeredTexture::~ResourceImporterLayeredTexture() {
 }
+
+void ResourceImporterLayeredTexture::_check_compress_stex(Ref<LayeredTextureImport> r_texture_import) {
+	String extension = get_save_extension();
+	ERR_FAIL_NULL(r_texture_import->csource);
+	if (r_texture_import->compress_mode != COMPRESS_VRAM_COMPRESSED) {
+		//import normally
```

Comment:
Review code style to match current conversion:
```
// Import normally.
```
(i.e. space after "//", capital letters, punctuation)

Same for other comments.